doc/distrib/NodeHelpFiles/_review/Revit.Elements.Parameter.SpecType.md

```diff
@@ -0,0 +1,16 @@
+## In Depth
+`Parameter.SpecType` returns the given parameter's SpecType. SpecType describes the type of data that a given parameter stores. 
+
+These data types include:
```

Comment:
There are more than the below listed spec types. So, to be clear suggest a minor modification as "These data type includes below (but are not limited to):Boolean,Int,String,e....etc"